### DISCUSSION TITLE
Lightning PyTorch Profiler

### DISCUSSION THREAD

**Discussion by austinmw** [torchgeo/torchgeo#835] [Q&A]
Hi, has anyone happened to try `Trainer(profiler='pytorch', ...)` and use it successfully with TorchGeo? I ask because it appears to result in a memory error and crash at the very end of training for me, but it might just be my setup (SageMaker). The `simple` and `advanced` profiler options work fine for me.

**Comment by calebrob6** [comment]
Hey @austinmw, I haven't tried that. Do you mind posting more details (I'm also generally unfamiliar with the Lightning profiler)?
